### DISCUSSION TITLE
NameError: name 'null' is not defined

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1725] [General Q&A]
got error after run that.--
!python3 '/content/drive/My Drive/MainTTS/train_glowtts.py'

**Comment by p0p4k** [comment]
Please post complete error logs. Your post is not informative enough to help you. Thanks.

**Comment by omkarade** [comment]
Thanks for reply! this error show.
![Screenshot (340)](https://user-images.githubusercontent.com/90675376/178570681-ddb3720f-cc71-4d1d-88b7-844e31b138f5.png)
